README.md

```diff
@@ -34,7 +34,7 @@ docker build . -t celestia-local-devnet
 To run the Docker container:
```

Comment:
on line 31, should it be `docker build . -t -i celestia-local-devnet` or without either flag `docker build . celestia-local-devnet`? @MSevey